lib/site-inspector/checks/accessibility.rb

```diff
@@ -0,0 +1,32 @@
+require 'json'
+
+class SiteInspector
+  class Endpoint
+    class Accessibility < Check
+      
+      def section508
+        pa11y('Section508')
+      end
+      
+      def wcag2a
+        pa11y('WCAG2A')
+      end
+      
+      def wcag2aa
+        pa11y('WCAG2AA')
+      end
+      
+      def wcag2aaa
+        pa11y('WCAG2AAA')
+      end
+      
+      private
+      
+      def pa11y(reporter)
+        json_string = `pa11y https://18f.gsa.gov -s #{reporter} -r json`
```

Comment:
> Last one (I hope), I'd verify that the reporter is in a list of allowed reporters, rather than passing arbitrary input to command line.

From the perspective of input sanitization and preventing injection attacks, strong agree.